.circleci/config.yml

```diff
@@ -25,29 +25,27 @@ jobs:
         paths:
         - venv/
         key: python-packages-v1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "constraints.txt" }}
-  verify-generated-sql:
+  verify-format-sql:
     docker:
       - image: python:3.8
     steps:
       - checkout
-      - *build
       - run:
-          name: Verify that all of generated SQL is committed
-          command: |
-            rm -rf sql/
-            venv/bin/python script/generate_sql
-            echo "Inconsistencies between templates and generated SQL:"
-            git ls-files --other --modified -x sql/*
-            test `git ls-files --other --modified -x sql/* | wc -l` = 0
-  verify-format-sql:
+          name: Verify that SQL is correctly formatted
+          # check all directories owned by git except sql
+          command: script/format_sql --check $(git ls-tree -d HEAD --name-only | grep -vx sql)
+  publish-persistent-udfs:
```

Comment:
This is failing, not sure what the best way for this is.

Reply:
We handle view publishing as part of the Jenkins job that does schema deploys (see https://github.com/mozilla-services/cloudops-infra/blob/9a7b3eb1b2ee7a3f068a7bb3cda626cc19dd1f72/projects/data-shared/Jenkinsfile.bigquery.prod#L100-L110).

I think we should publish UDFs at that same point, right before we publish views.

We'll still have the problem that PR authors will likely need to manually publish changes in order to get the dry run CI task passing, which is currently the case with PRs that add new queries.